src/yunohost/app.py

```diff
@@ -52,8 +52,13 @@
     mkdir,
 )
 
-from yunohost.service import service_status, _run_service_command
 from yunohost.utils import packages
+from yunohost.utils.config import (
+    ConfigPanel,
+    parse_args_in_yunohost_format,
+    Question,
```

Comment:
I don't understand why you moved `parse_args_in_yunohost_format` and `Question` into the config.py file since it's related to app install, upgrade, etc...

At least the app.py file is shorter now

Reply:
Because now Question and ConfigPanel are used to create the commands `yunohost domain config get|set` too.

And because the app.py is too looooooong and it's boring to navigate between 4000+ lines.